Rakefile

```diff
@@ -1,7 +1,17 @@
 require 'bundler/gem_tasks'
 require 'rake/testtask'
 
+require 'news_scraper'
+
 Rake::TestTask.new do |t|
   t.libs << 'test'
   t.pattern = 'test/**/*_test.rb'
 end
+
+namespace :scraper do
+  desc 'CLI that steps through articles for a given query and displays preset scrape pattern results; parameters: QUERY'
+  task :train do
+    query = ENV['QUERY'] || ''
```

Comment:
We should raise if the query is not there.
```
query param was not given.
Usage:
  bundle exec rake scraper:train QUERY=<query>
```

Also, `ENV.fetch('QUERY', '')` is the preferred way to do `ENV || ''`